### PR TITLE
[FLINK-2741] - Use single log statement in TestLogger

### PR DESCRIPTION
**Pull request by rerngvit**
[FLINK-2741] - Use single log statement in TestLogger

### CODE REVIEW

**Comment by hsaputra** [comment]
Unfortunately looks like this patch does not take advantage of slf4j parameterized logging [1].

Either do like:

log.info("=================================" + "\nTest " + description + " is running." + "\n--------------------------------------------------------------------------------");
or:

log.info("===============================" + "\nTest {} is running." + "\n--------------------------------------------------------------------------------", description);


[1] http://www.slf4j.org/faq.html#logging_performance

**Comment by rerngvit** [comment]
@hsaputra Thanks for your review. I modified this PR according to your comment.

**Comment by rerngvit** [comment]
The build is failed on an issue not related to the PR.
In particular, it failed only on "JDK8 Hadoop 2.5" with
"java.io.FileNotFoundException: build-target/lib/flink-dist-*.jar "

**Comment by hsaputra** [comment]
+1

**Comment by hsaputra** [comment]
Thanks for the patch update. LGTM

**Comment by chiwanpark** [comment]
I'm not sure this patch is ready to merge. Because of additional components for log such as time, thread name, caller class name, etc., the log output is not same after merging this changes.

I'm inclined to preserve the current implementation.

**Comment by hsaputra** [comment]
Thanks for second pair of eyes :)
How the output result is different? 
The patch just make sure logging of all messages happen in single log call

File: flink-core/src/test/java/org/apache/flink/util/TestLogger.java [hunk 1]
```diff
@@ -37,23 +37,26 @@
 
 		@Override
 		public void starting(Description description) {
-			log.info("================================================================================");
-			log.info("Test {} is running.", description);
-			log.info("--------------------------------------------------------------------------------");
+			log.info("================================================================================"
```
Comment (uce) [inline flink-core/src/test/java/org/apache/flink/util/TestLogger.java]: I would a new line here before the line as well. Then we have everything together with the same indentation.

**Comment by StephanEwen** [comment]
The alignment gets thrown off. You can correct this with Ufuk's hint, adding a newline at the very beginning.

**Comment by rerngvit** [comment]
@uce I added the new line according to your hint.

**Comment by chiwanpark** [comment]
I also meant the alignment. You need more newlines for `succeeded` and `failed`.

**Comment by uce** [comment]
Thanks :). I'm going to merge this (I will insert the missing line breaks).

**Comment by rerngvit** [comment]
Thanks all for suggestions.

**Pull request closed without merging**